### PR TITLE
Add FreeBayes bamleftalign

### PR DESCRIPTION
**Pull request by ihodes**
fixes https://github.com/hammerlab/biokepi/issues/390

(doesn't add the variant caller,  to the workflow but all the infrastructure is there)

### CODE REVIEW

File: src/environment_setup/tool_providers.ml [hunk 1]
```diff
@@ -116,23 +189,36 @@ let witness_list l =
...
     Machine.Tool.Default.bwa
     ~url:"http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.10.tar.bz2"
     ~install_program:(make_and_copy_bin "bwa")
     ~init_program:add_to_dollar_path
     ~witness:(witness_file "bwa")
 
+let freebayes =
+  Git_installable_tool.make
+    Machine.Tool.Default.freebayes
+    ~repository:"git://github.com/ekg/freebayes.git"
```
Comment (smondet) [inline src/environment_setup/tool_providers.ml]: Maybe the `https://` version is more portable (against firewalls & co).

File: src/pipeline_edsl/semantics.ml [hunk 2]
```diff
@@ -171,6 +171,11 @@ module type Bioinformatics_base = sig
     [ `Bed ] repr ->
     [ `Vcf ] repr
 
+  val bam_left_align:
```
Comment (smondet) [inline src/pipeline_edsl/semantics.ml]: Here (in `Semantics`), I like adding doc-comments progressively for new tools.

File: tools/docker/run/Dockerfile [hunk 3]
```diff
@@ -5,7 +5,7 @@
 FROM hammerlab/ketrew-server
 # Installing easy Biokepi dependencies:
 RUN sudo apt-get update
-RUN sudo apt-get install --yes cmake r-base tcsh libx11-dev libfreetype6-dev pkg-config wget gawk graphviz xvfb
+RUN sudo apt-get install --yes cmake r-base tcsh libx11-dev libfreetype6-dev pkg-config wget gawk graphviz xvfb git
```
Comment (smondet) [inline tools/docker/run/Dockerfile]: Can you add the corresponding line in `tools/docker/README.md`?

**Comment by ihodes** [comment]
@smondet back to you, thanks for the review!

**Pull request merged**